dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/validation/field/validators/StandardBooleanValidator.java

```diff
@@ -0,0 +1,51 @@
+package edu.harvard.iq.dataverse.validation.field.validators;
+
+import edu.harvard.iq.dataverse.common.BundleUtil;
+import edu.harvard.iq.dataverse.persistence.dataset.ValidatableField;
+import edu.harvard.iq.dataverse.validation.field.FieldValidationResult;
+import org.omnifaces.cdi.Eager;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.List;
+import java.util.Map;
+
+@Eager
+@ApplicationScoped
+public class StandardBooleanValidator extends MultiValueValidatorBase {
+    public static final String ACCEPTABLE_VALUE_KEY = "acceptableValue";
+
+    @Override
+    public String getName() {
+        return "standard_boolean";
+    }
+
+    @Override
+    public FieldValidationResult validateValue(String value, ValidatableField field, Map<String, Object> params,
+                                               Map<String, ? extends List<? extends ValidatableField>> fieldIndex) {
+
+        Object acceptableValue = params.get(ACCEPTABLE_VALUE_KEY);
+        if(acceptableValue == null) {
```

Comment:
I am not 100% sure, but at first sight, it seems that this if is not needed, because the following ifs are covering this case.

Reply:
It is needed, as if there is no default acceptable value, then it will verify for true value.